.github/workflows/maven.yml

```diff
@@ -50,15 +50,13 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-      - name: Append 'SNAPSHOT' to version if 'push' happened on 'main'
-        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
-        run: |
-          mvn help:evaluate -N -Dexpression=project.version|grep -v '\['
-          VERSION=$(mvn help:evaluate -N -Dexpression=project.version|grep -v '\[')
-          mvn versions:set -DnewVersion=$VERSION-SNAPSHOT
+      - name: Remove "-SNAPSHOT" if 'push' happened on a tag
```

Comment:
I'm not sure. Putting this here does nothing if the SNAPSHOT was removed though.

Reply:
That will mean that the last commit does include SNAPSHOT and do not match what you are building